.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   root: true,
-  extends: "babel",
+  parser: "lightscript-eslint",
```

Comment:
Yeah, I found that worked for me. `eslint-config-babel` depends on `babel-eslint` as a peerDependency. They're both installed as devdeps.